dev-python/pytest-asyncio/pytest-asyncio-0.15.1.ebuild

```diff
@@ -23,4 +23,11 @@ BDEPEND="
 		>=dev-python/hypothesis-3.64[${PYTHON_USEDEP}]
 	)"
 
+python_prepare_all() {
+	# Ignore deprecation warnings
+	sed -i -E 's/^filterwarnings = error/&\n    ignore::DeprecationWarning/' setup.cfg || die
```

Comment:
I'd rather not do that, unless upstream actually blesses it. Then, it would be more readable as a patch.

Reply:
Hmm, I see upstream's working hard on a major change, so I suppose this is fine after all.

Reply:
Should I still make it a patch instead?

Reply:
Yes, please.